exercises/dominoes/dominoes.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Dominoes do
+
+  @doc """
+  chain?/1 takes a list of integer and returns boolean indicating if chain is possible
+  number to 1 when following the rules:
+    - if number is odd, multiply with 3 and add 1
+    - if number is even, divide by 2
+  """
+  @spec chain?(dominoes :: list) :: boolean
```

Comment:
Hmmm, I just sort of assumed you could do that, but I guess you can only define that type as `[1..6]`, or we could use the tuple. Personally, I sort of like the tuple, but I'm cool with just `[1..6]` if you don't want to have to re-write this. It's not that big of a deal.